misc/toml-parser/src/main/java/io/ballerina/toml/validator/SchemaValidator.java

```diff
@@ -88,26 +89,43 @@ public void visit(TomlTableNode tomlTableNode) {
                 DiagnosticInfo diagnosticInfo = new DiagnosticInfo("TVE0001", "error.unexpected.property",
                         DiagnosticSeverity.ERROR);
                 TomlDiagnostic diagnostic = new TomlDiagnostic(value.location(), diagnosticInfo,
-                        "Unexpected Property \"" + key + "\"");
+                        getUnexpectedPropertyErrorMessage(key));
                 tomlTableNode.addDiagnostic(diagnostic);
             }
         }
         for (String field : requiredFields) {
             DiagnosticInfo diagnosticInfo = new DiagnosticInfo("TVE0006", "error.required.field.missing",
                     DiagnosticSeverity.ERROR);
             TomlDiagnostic diagnostic = new TomlDiagnostic(tomlTableNode.location(), diagnosticInfo,
-                    "Missing required field \"" + field + "\"");
+                    getRequiredErrorMessage(field));
             tomlTableNode.addDiagnostic(diagnostic);
         }
     }
 
+    private String getRequiredErrorMessage(String field) {
+        Map<String, String> message = this.schema.message();
+        String typeCustomMessage = message.get(SchemaDeserializer.REQUIRED);
+        if (typeCustomMessage == null) {
+            return "Missing required field \"" + field + "\"";
+        }
+        return typeCustomMessage.replace(PROPERTY_HOLDER, field);
+    }
+
+    private String getUnexpectedPropertyErrorMessage(String key) {
+        Map<String, String> message = this.schema.message();
+        String typeCustomMessage = message.get(SchemaDeserializer.ADDITIONAL_PROPERTIES);
+        if (typeCustomMessage == null) {
+            return "Unexpected Property \"" + key + "\"";
+        }
+        return typeCustomMessage.replace(PROPERTY_HOLDER, key);
+    }
```

Comment:
These errors get printed in the format you mentioned right? I think we need to revisit the format and phrasing of these sort of error messages to align with the other compiler/parser error messages. e.g., start with lower case letters, using single quotes to quote keys/fields etc.

Reply:
@MaryamZi

Reply:
@pubudu91 @MaryamZi  I'm currently integrating this toml parser with schema validation to replace the validations we do in the `ManifestBuilder` class. I will mention you in the PR, once I'm done with it.

Reply:
@pubudu91 Yes. removed caps and double quotes from all the default error messages and test cases in the recent commit.